pages/components/button.html

```diff
@@ -0,0 +1,82 @@
+<html>
+  <head>
+    <link href="../../styles/style.css" rel="stylesheet" />
+    <link href="../../styles/drop.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Dropcss</title>
+  </head>
+  <body>
+    <navBar>
+      <div class="navbar">
+        <a href="../../index.html">
+          <p class="font-size-4xl">DropCss</p>
+        </a>
+      </div>
+    </navBar>
+    <div class="flex">
+      <aside class="aside">
+        <div>
+          <p class="font-size-lg text-semibold">Getting Started</p>
+          <ul class="components-list">
+            <li class="text-medium text-gray2 font-size-lg">
+              <a href="getStarted.html">Installation</a>
+            </li>
+          </ul>
+          <p class="font-size-lg text-semibold">Components</p>
+          <ul class="components-list font-size-lg text-medium text-gray2">
+            <li><a href="./alert.html">Alert</a></li>
+            <li><a href="./avatar.html">Avatar</a></li>
+            <li>Badge</li>
+            <li><a href="./button.html" class="active">Button</a></li>
+            <li>Card</li>
+            <li>Image</li>
+            <li>Input</li>
+            <li>List</li>
+            <li>Text Utilities</li>
+            <li>Navigation</li>
+            <li>Modal</li>
+            <li>Toast</li>
+            <li>Rating</li>
+            <li>Slider</li>
+            <li>Grid</li>
+          </ul>
+        </div>
+      </aside>
+      <main class="main">
+        <div>
+          <p class="text-gray2 text-medium font-size-2xl">Button</p>
+          <p class="padding-top-4 font-size-lg text-gray line-height">
+            Buttons allow an user to take an action like triggering an event,
+            opening/closing an application, submitting a form etc.
+          </p>
+          <div class="btn-container flex justify-around margin-top-8">
+            <button class="btn btn-primary">Primary</button>
+            <button class="btn btn-secondary">Secondary</button>
+            <button class="btn btn-outline-primary">Primary</button>
+            <button class="btn btn-outline-secondary">Secondary</button>
+            <button class="btn btn-icon">
+              <i class="btn-icon-in fa fa-home"></i>Home
+            </button>
```

Comment:
- You can provide cursor as pointer for buttons.
- Else looks good to me nice work.